apps/galaxy/src/services/signin/useSignIn.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "../utils/axios";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { useToast } from "@cc/ui-chakra";
+import { AxiosError } from "axios";
+
+export default function useSignIn() {
+  const [isLoading, setIsLoading] = useState(false);
+  const toast = useToast();
+  const router = useRouter();
+  const handleSignInSubmit = async (username: string, password: string) => {
+    setIsLoading(true);
+    try {
+      const response = await axios.post("/staff/signin", {
+        username: username,
+        password: password,
+      });
+      if (response.status === 200) {
+        router.push("/");
+      }
+    } catch (error) {
+      const err = error as AxiosError;
```

Comment:
Just curious, why is why are re-declaring `err` objects? if this is just to cast the `error` to `AxiosError`, you can also do `(error as AxiosError).response`. 

Another approach I recommend is to just deconstruct `{response, message} = error as AxiosError`...

Reply:
There was an error when I tried to use `(error as AxiosError).response`,Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196).

`{response, message} = error as AxiosError` I changed to this deconstuct approach